include/CMakeLists.txt

```diff
@@ -1 +1,30 @@
 add_subdirectory(swift)
+
+
+# Create a library that depends on all headers defined in include/swift/module.modulemap
+#
+# TODO: generate this dynamically through the modulemap; this cannot use `sed`
+# as that is not available on all platforms (e.g. Windows).
+file(GENERATE
```

Comment:
`file(GENERATE` only updates the file when the content has changed. We don't need the `.tmp`/`add_custom_command` hack.